lib/websocket/websocket-handler.ts

```diff
@@ -233,9 +240,7 @@ function setup(socket: Interface.ISocket): void
             activeSubscriptions.delete(s.correlationId);
             receivedSubscriptions.delete(s.correlationId);
         });
-        receivedSubscriptions.size
-            ? socket.send('unsubscribed')
-            : socket.send('unsubscribed all');
+        socket.notifyUnsubscribed(0 === receivedSubscriptions.size);
```

Comment:
I guess we can simplify it in `socket.notifyUnsubscribe(!receivedSubscriptions.size)`